### PR TITLE
Create sam-jspn.json

### PR DESCRIPTION
**Pull request by Samuel-Jspn**

### CODE REVIEW

File: domains/sam-jspn.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Sam's website",
+  "owner": {
+    "username": "Sam-Jspn",
+    "email": "sam.josephine@outlook.fr",
+    "discord": "R3dL1ghty#2904"
+  },
+  "record": {
+    "CNAME": "sam-jspn.github.io"
```
Comment (MaskDuck) [inline domains/sam-jspn.json:9]: 404?

**Review by MaskDuck — CHANGES_REQUESTED** [review]

File: domains/sam-jspn.json (hunk 1 above)
Comment (Samuel-Jspn) [reply domains/sam-jspn.json]: resolved (i think so)

File: domains/sam-jspn.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Sam's website",
+  "owner": {
+    "username": "Sam-Jspn",
+    "email": "sam.josephine@outlook.fr",
+    "discord": "R3dL1ghty#2904"
+  },
+  "record": {
+    "CNAME": "https://samuel-jspn.github.io/lair/"
```
Comment (MaskDuck) [inline domains/sam-jspn.json:9]: cname shouldnt contain slash

**Review by MaskDuck — CHANGES_REQUESTED** [review]

File: domains/sam-jspn.json (hunk 2 above)
Comment (Samuel-Jspn) [reply domains/sam-jspn.json:9]: should it just be "samuel-jspn.github.io/lair" ? Or no slash at all ?

Comment (MaskDuck) [reply domains/sam-jspn.json]: no slash.

**Comment by MaskDuck** [comment]
@Samuel-Jspn your website *will* be accessible at https://sam-jspn.is-a.dev/lair , not https://sam-jspn.is-a.dev .
confirm?

**Comment by Samuel-Jspn** [comment]
> @Samuel-Jspn your website _will_ be accessible at https://sam-jspn.is-a.dev/lair , not https://sam-jspn.is-a.dev . confirm?

Confirmed

**Review by MaskDuck — APPROVED** [review]
you're welcome to join our discord server too, link included in readme!!!

**Pull request merged**